docs/developers/quickstart/postman-sdk.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Linea SDK - Postman
+sidebar_position: 2
+---
+
+## Introduction to Postman
+
+The Postman client is a part of the Linea SDK that dapp developers can use to integrate Linea's cross-chain message transfer process into their own infrastructure. Postman allows dapps to move messages from one chain to another and are responsible for those messages being executed. In return, the postman gets to collect a fee once it has been verified that the message has been delivered and recorded. In other words, this package lets your dapp be the postman for L1 and L2 messages, using Linea's protocol.
+
+> _Note: Fees are currently not setup for this postman integration but will be available in future iterations._
+
+## Installation
+
+To install the package you need to run:
+
+```bash
+npm install @consensys/linea-sdk
+```
+
+## Implementation
+
+The code below will allow your dapp to act as a Postman.
+
+```typescript
+import { PostmanServiceClient } from "@consensys/linea-sdk";
+import { format, transports } from "winston";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+async function main() {
+  const client = new PostmanServiceClient({
+    l1Config: {
```

Comment:
Do we have documentation on what each of these parameters are

Reply:
the .env file explains what all the parameters are if that's what you are asking?